recipe/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "enum-tools" %}
+{% set version = "0.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
+  sha256: 0d13335e361d300dc0f8fd82c8cf9951417246f9676144f5ee1761eb690228eb
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<37]
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
```

Comment:
```suggestion
```
Whey has its own wheel builder https://github.com/repo-helper/whey/blob/master/whey/builder.py#L631.
setuptools are not needed https://github.com/search?q=repo%3Adomdfcoding%2Fenum_tools%20setuptools&type=code